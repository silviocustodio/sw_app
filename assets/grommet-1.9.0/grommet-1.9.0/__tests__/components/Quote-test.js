// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react-test-renderer';

import Quote from '../../src/js/components/Quote';

// needed because this:
// https://github.com/facebook/jest/issues/1353
jest.mock('react-dom');

describe('Quote', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <Quote>Testing</Quote>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
