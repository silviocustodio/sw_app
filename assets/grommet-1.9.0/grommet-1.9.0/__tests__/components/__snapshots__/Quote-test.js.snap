// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote has correct default options 1`] = `
<div
  className="grommetux-box grommetux-box--direction-column grommetux-box--responsive grommetux-box--pad-horizontal-large grommetux-box--pad-vertical-small grommetux-box--size-large grommetux-box--size grommetux-quote grommetux-quote--emphasize-credit"
  id={undefined}
  onClick={undefined}
  role={undefined}
  style={Object {}}
  tabIndex={undefined}
>
  <div>
    Testing
  </div>
</div>
`;
